model/Aktivita/Aktivita.php

```diff
@@ -2366,6 +2366,52 @@ public static function prihlasovatkoZpracuj(?Uzivatel $u, ?Uzivatel $prihlasujic
         }
     }
 
+    /** Zpracuje post data z přihlašovátka. */
+    public static function prihlasovatkoZpracujJSON(?Uzivatel $u, ?Uzivatel $prihlasujici, $parametry = 0)
```

Comment:
Tohle prosím nedělejme, to je cesta do pekel :smiling_imp: 

Jediný rozdíl mezi `prihlasovatkoZpracuj` a touhle metodou je, že původní má v sobě `back`, jinak je logika stejná a my bychom si tímhle dost zavařili, protože bychom museli udržovat stejnou logiku na více místech (zažil jsem to už hodněkrát a nakonec to způsobí zhroucení projektu, který už nepůjde rozumně upravovat bez velkých chyb).

Navrhuju udělat jednu private metodu, která místo `back` bude vracet true a jinak false. A dvě public metody, které si po svém zareagují na to, že se něco změnilo:

```php
public static function prihlasovatkoZpracuj(...) {
  if (self::prihlasovatkoZpracujData(...)) {
    back();
  }
}

public static function prihlasovatkoZpracujJSON(...) {
  self::prihlasovatkoZpracujData(...);
}
```

:warning: je tu jedna změna v logice, která nevím jestli je schválně. V originále je `back`, které okamžitě způsobí redirect a **exit**, takže poslední logika se nespustí, pokud se něco zpracovalo
```php
        if ($parametry & self::PLUSMINUS_KAZDY) {
            self::plusminusZpracuj();
        }
```
ale v nové metodě se to **spustí vždy**. Je to OK?